### PR TITLE
Qt6 port

### PR DESCRIPTION
**Pull request by tsujan**

### CODE REVIEW

File: lxqt-session/src/lxqtmodman.cpp [hunk 1]
```diff
@@ -190,9 +188,11 @@ void LXQtModuleManager::startWm(LXQt::Settings *settings)
 {
     // if the WM is active do not run WM.
     // all window managers must set their name according to the spec
-    if (!QString::fromUtf8(NETRootInfo(QX11Info::connection(), NET::SupportingWMCheck).wmName()).isEmpty())
-    {
-        return;
+    if (auto x11NativeInterfce = qGuiApp->nativeInterface<QNativeInterface::QX11Application>()) {
+        if (!QString::fromUtf8(NETRootInfo(x11NativeInterfce->connection(), NET::SupportingWMCheck).wmName()).isEmpty())
```
Comment (stefonarch) [inline lxqt-session/src/lxqtmodman.cpp:192]: `11NativeInterfce->connection()` typo?

Comment (tsujan) [reply lxqt-session/src/lxqtmodman.cpp:192]: No, it isn't a typo. Why should it be?

Comment (Chiitoo) [reply lxqt-session/src/lxqtmodman.cpp:192]: The `Interfce` does look a bit odd fer sure.

Comment (tsujan) [reply lxqt-session/src/lxqtmodman.cpp:192]: This is how it's done in Qt6. They got rid of `qt5-x11extras` and `QX11Info`.

Comment (Chiitoo) [reply lxqt-session/src/lxqtmodman.cpp:192]: Was referring to the spelling `Interfce` instead of `Interface`.  : ]

Comment (tsujan) [reply lxqt-session/src/lxqtmodman.cpp:192]: LOL! Why didn't you say it the first time? I thought you both meant _the code_. It's a known fact that the brain corrects such typos while reading ;)

Thanks! I'll correct them; I copy-pasted them from libfm-qt; so I'll correct them there too.

Comment (Chiitoo) [reply lxqt-session/src/lxqtmodman.cpp:192]: Heh, yeah, my intention was to be very clear about it but realised it a bit too late that I really wasn't... sorry about that!

**Pull request merged**